code/modules/interface/input.dm

```diff
@@ -175,7 +175,7 @@ var/list/dirty_keystates = list()
 
 		if(findtext( control, "viewport" ))
 			var/datum/viewport/vp = getViewportById(control)
-			if(vp && vp.clickToMove && object && isturf(object) && (mob.type == /mob/living/intangible/blob_overmind || mob.type == /mob/dead/aieye))//NYI: Replace the typechecks with something Better.
+			if(vp && vp.clickToMove && object && isturf(object) && (mob.type == /mob/living/intangible/blob_overmind || mob.type == /mob/living/intangible/aieye))//NYI: Replace the typechecks with something Better.
```

Comment:
istype() preferred

Reply:
should probably just replace` (mob.type == /mob/living/intangible/blob_overmind || mob.type == /mob/living/intangible/aieye))`
with `isintangible(mob)`